community.mdwn

```diff
@@ -44,4 +44,4 @@ You can help by:
 - [Flattr](https://flattr.com/thing/48025/awesome-window-manager)
 
 # Authors
-[[!inline pages="authors" raw="yes" rss="no"]]
+[See out GitHub page](https://github.com/awesomeWM/awesome/graphs/contributors)
```

Comment:
What do you mean with "it is broken"? The page looks fine here:
https://awesomewm.org/community/

(Perhaps not all that up-to-date, but that has other reasons)